### ISSUE TITLE
GPS Track resets when phone falls asleep

### ISSUE THREAD

**Issue by alecgirman** [mracko/MSFS-Mobile-Companion-App#15]
Aircraft: Flybywire A320NX

The 'GPS Track' option (rendering the red line behind the aircraft) resets when my phone falls asleep.  Likewise,  the red line goes from where it was when I last woke up my phone to my current location, so I think its streaming the data to the app and not saving a log on the client (PC running MSFS).  I think it will be nice to have a persistent GPS track of the aircraft.

**Comment by mracko** [comment]
You're right, the GPS track line is handled entirely by the client ie your mobile device. This means that when you refresh the website, turn off your mobile device or anything similar, the track line is gone. I use No Screen Off on my mobile devices to make sure that my screen doesn't get dark.

I'll have a look at changing the app so that the track line is handled by the server.
